lib/CommentThread.php

```diff
@@ -54,6 +91,19 @@ public function order( $order = 'ASC' ) {
 		return $this;
 	}
 
+	/**
+	 * @internal
+	 * @experimental
```

Comment:
The `@experimental` tag is not really an official tag, or is it?

Reply:
No, those are something I saw at some point and brought over — but not official. I think we should scrub them all (and probably the associated functions in 2.0)

Reply:
I just added a [project card](https://github.com/timber/timber/projects/4#card-22187846) for this.